python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/flow_tracking.schema.yml

```
@@ -72,3 +72,87 @@ keys:
               format_ipfix_standard_timestamps_counters:
                 type: bool
                 description: Enable software export of IPFIX data records.
+      mirror_on_drop:
+        type: dict
+        keys:
+          encapsulation:
+            type: dict
+            keys:
+              ipv4_ipv6:
+                description: |-
+                 Set IPv4 and IPv6 encapsulations.
+                 Both IPv4 and IPv6 encapsulations must be set together.
+                type: bool
+              mpls:
+                type: bool
+          sample_limit:
+            description: Limit the number of packets sampled.
+            type: int
+            convert_types:
+              - str
+            min: 1
+            max: 4294967295
+          trackers:
+            type: list
+            primary_key: name
+            items:
+              type: dict
+              keys:
+                name:
+                  type: str
+                  description: Tracker Name.
+                record_export:
+                  type: dict
+                  $ref: "eos_cli_config_gen#/$defs/flow_tracking/keys/trackers/items/keys/record_export"
+                exporters:
+                  type: list
+                  primary_key: name
+                  items:
+                    type: dict
+                    keys:
+                      name:
+                        type: str
+                        description: Exporter Name.
+                      collectors:
+                        type: list
+                        primary_key: name
+                        items:
+                          type: dict
+                          keys:
+                            name:
+                              type: str
+                              description: |-
+                                Flow collector name.
+                                It can be IPv4 address, IPv6 address, fully qualified domain name or sflow.
```

Do you mean the string "sflow" ? 
```suggestion
                                The collector name can be an IPv4 address, IPv6 address, fully qualified domain name or `"sflow"`.
```

---

Could it be more intuitive if we had `collector_sflow` as a boolean besides this list?

---

In the other flow_tracking schema we use `host` as the key. Please see if you can use more $refs here so we don't have to redefine everything here.

---

About collector_sflow yes we can but we can add port number along with it so we need to create a dict instead of bool. In that case we can keep it under collectors[].host as sflow

---

About using more $refs then in some cases the structure config sequence getting change and we created a task #5502 to handle the all refs.